src/components/PastTripNotes.js

```diff
@@ -1,24 +1,44 @@
 import styled from 'styled-components';
 import { FiTrash } from 'react-icons/fi';
+import Modal from './Modal';
+import { useState } from 'react';
+
+export default function PastTripNotes({ notes, onDelete }) {
+  const [visible, setVisible] = useState(false);
+  const [id, setId] = useState('');
```

Comment:
I don't quite understand this 😬 Why do you need the state when you can just pass the `id` directly?